module-domain/src/main/kotlin/site/yourevents/guest/port/out/GuestPersistencePort.kt

```diff
@@ -20,4 +20,6 @@ interface GuestPersistencePort {
     fun findAttendGuestsByInvitation(invitation: Invitation): List<Guest>
 
     fun findNotAttendGuestsByInvitation(invitation: Invitation): List<Guest>
+
+    fun findByMemberAndInvitation(memberId: UUID, invitationId: UUID): Boolean?
```

Comment:
출석 필드를 찾아서 반환하는데, 해당 메서드명은 무엇을 반환하는지 명시가 되어있지 않아 조금 혼란스러울 수 있을 것 같아 보입니다..!
```suggestion
    fun findAttendanceByMemberAndInvitation(memberId: UUID, invitationId: UUID): Boolean?
```

다음과 같이 수정하면 명료성을 높일 수 있지 않을까요?

Reply:
넵!! 정완님 의견이 맞는 것 같습니다!! 수정하였습니다.